src/mount.ts

```diff
@@ -19,13 +26,17 @@ export type MountOptions = {
  * The component can be unmounted by calling `wrapper.unmount()` or by calling
  * {@link unmountAll} at the end of the test.
  */
-export function mount(jsx: VNode, { connected = false }: MountOptions = {}) {
+export function mount(
+  jsx: VNode,
+  { connected = false, prepareContainer }: MountOptions = {},
+) {
   let wrapper;
   if (connected) {
     const container = document.createElement('div');
     container.setAttribute('data-enzyme-container', '');
     containers.push(container);
 
+    prepareContainer?.(container);
```

Comment:
For now I added the call before appending the container to the body, because that was the approach in all places where we were setting styles.

But yeah, it's possible we need to do it the other way around in the future.